Godeps/Godeps.json

```diff
@@ -1,10 +1,21 @@
 {
 	"ImportPath": "github.com/maruel/panicparse",
-	"GoVersion": "go1.4.2",
+	"GoVersion": "devel +adf4c38 Thu Nov 26 17:42:14 2015 +0000",
```

Comment:
I'd prefer it to be a real go toolset version, e.g. 1.5.1. But I don't know why this line is embedded at all, this has no value.